.github/workflows/bot-command.yml

```
@@ -0,0 +1,86 @@
+  issues: write
+  contents: read
+
+jobs:
+  Bot-command-check:
+    name: Bot command check
+    if: github.event_name == 'pull_request' || startsWith(github.event.comment.body, '/bot')
+    runs-on: ubuntu-latest
+    outputs:
+      status: ${{ steps.check.outputs.status }}
+    steps:
+      - id: check
+        name: Check bot command
+        run: |
+          if [[ "${{ github.event.comment.body }}" == "/bot run"* ]] ||
+             [[ "${{ github.event.comment.body }}" == "/bot skip --comment"* ]] ||
+             [[ "${{ github.event.comment.body }}" == "/bot reuse-pipeline"* ]] ||
+             [[ "${{ github.event.comment.body }}" == "/bot kill"* ]]; then
+            echo 'status=success' >> $GITHUB_OUTPUT
+          else
+            echo 'status=failure' >> $GITHUB_OUTPUT
+          fi
+      - name: Add bot help comment
+        if: steps.check.outputs.status == 'failure'
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const helpMessage = "" +
+            "## GitHub Bot Help\n\n" +
+            "`/bot [-h] ['run', 'kill', 'skip', 'reuse-pipeline'] ...`\n\n" +
+            "provides a user friendly way for developers to interact with a Jenkins server.\n\n" +
+            "Run `/bot [-h|--help]` to print this help message.\n\n" +
+            "See details below for each supported subcommand.\n\n" +
+            "<details>\n\n" +
+            "`run  [--disable-fail-fast  --skip-test  --stage-list \"A10-e2e-0, xxx\" --gpu-type \"A30, H100_PCIe\" --multi-gpu-test  --post-merge  --extra-stage \"H100_PCIe-[Post-Merge]accuracy-0, xxx\" --memory-profiling  --disable-incremental-build  --enable-publish-last-known-good ]`\n\n" +
+            "Launch build/test pipelines. All previously running jobs will be killed.\n\n" +
+            "`--skip-test ` *(OPTIONAL)* : Skip all test stages, but still run build stages, package stages and sanity check stages. Note: Does **NOT** update GitHub check status.\n\n" +
```

Please add `--disable-fail-fast  (OPTIONAL) : Disable fail fast on build/tests/infra failures.` as well.

---

Done